tasks/test_application.rb

```diff
@@ -18,6 +18,7 @@ def soft_generate
       end
       Bundler.with_original_env do
         Kernel.system("yarn install") # so tailwindcss/plugin is available for test app
+        Kernel.system("rake dependencies:vendor") # so flowbite is available for test app
```

Comment:
This would be available regardless (different reason from the line comment above), we just want to ensure the file is updated from the installed version in our npm package dependencies.

```suggestion
        Kernel.system("rake dependencies:vendor") # ensure flowbite is updated for test app
```

Reply:
Thanks! Done